picard/ui/options/renaming_compat.py

```diff
@@ -0,0 +1,215 @@
+# -*- coding: utf-8 -*-
+#
+# Picard, the next-generation MusicBrainz tagger
+#
+# Copyright (C) 2006-2008, 2011 Lukáš Lalinský
+# Copyright (C) 2008-2009 Nikolai Prokoschenko
+# Copyright (C) 2009-2010, 2014-2015, 2018-2022 Philipp Wolfer
+# Copyright (C) 2011-2013 Michael Wiencek
+# Copyright (C) 2011-2013 Wieland Hoffmann
+# Copyright (C) 2013 Calvin Walton
+# Copyright (C) 2013 Ionuț Ciocîrlan
+# Copyright (C) 2013-2014 Sophist-UK
+# Copyright (C) 2013-2015, 2018-2021 Laurent Monin
+# Copyright (C) 2015 Alex Berman
+# Copyright (C) 2015 Ohm Patel
+# Copyright (C) 2016 Suhas
+# Copyright (C) 2016-2017 Sambhav Kothari
+# Copyright (C) 2021 Bob Swift
+# Copyright (C) 2021 Gabriel Ferreira
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+
+import re
+
+from PyQt5 import (
+    QtCore,
+    QtGui,
+    QtWidgets,
+)
+
+from picard.config import (
+    BoolOption,
+    Option,
+    get_config,
+)
+from picard.const.sys import IS_WIN
+from picard.util import system_supports_long_paths
+
+from picard.ui import PicardDialog
+from picard.ui.options import (
+    OptionsPage,
+    register_options_page,
+)
+from picard.ui.ui_options_renaming_compat import Ui_RenamingCompatOptionsPage
+from picard.ui.ui_win_compat_dialog import Ui_WinCompatDialog
+
+
+class RenamingCompatOptionsPage(OptionsPage):
+
+    NAME = "filerenaming_compat"
+    TITLE = N_("Compatibility")
+    PARENT = "filerenaming"
+    ACTIVE = True
+    HELP_URL = '/config/options_filerenaming_compat.html'
+
+    options = [
+        BoolOption("setting", "windows_compatibility", True),
+        BoolOption("setting", "windows_long_paths", system_supports_long_paths() if IS_WIN else False),
+        BoolOption("setting", "ascii_filenames", False),
+        BoolOption("setting", "replace_spaces_with_underscores", False),
+        Option("setting", "win_compat_replacements", {
+            '*': '_',
```

Comment:
https://github.com/metabrainz/picard/pull/2177/files#diff-9fd922f0357a6d93bbfac9746d8a3f164d73a879a6b68923fd0f5af83bd148f8R205 defines a local variable `default`, we could use a class level constant and use it here too.